pkg/controller/route/routecontroller.go

```diff
@@ -87,6 +88,12 @@ func (rc *RouteController) reconcileNodeRoutes() error {
 	return rc.reconcile(nodeList.Items, routeList)
 }
 
+const alreadyExistError = "googleapi: Error 409"
```

Comment:
I know this is a workaround for GCE. Just feel uncomfortable to have googleapi to this generic error: alreadyExistError.